### PR TITLE
graphviz, bump version

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
@threedeyes there were some patches from you in the previous version, I went to look for them but couldn't find the files that were used there (I guess they solved/or changed some things in that regard).  Could you check?
PS, checked with the latest Cutter release and it's fine there.

File: media-gfx/graphviz/graphviz-2.49.1.recipe [hunk 1]
```diff
@@ -12,11 +12,11 @@ Graphviz has many useful features for concrete diagrams, such as options for \
...
-COPYRIGHT="1994-2020 AT&T Corp."
+COPYRIGHT="1994-2021 AT&T Corp."
 LICENSE="EPL v1.0"
-REVISION="3"
-SOURCE_URI="https://www2.graphviz.org/Packages/stable/portable_source/graphviz-$portVersion.tar.gz"
-CHECKSUM_SHA256="9aabd13a8018b708ab3c822de2326c19d0a52ed59f50a6b0f9318c07e2a6d93b"
+REVISION="1"
+SOURCE_URI="https://gitlab.com/graphviz/graphviz/-/archive/$portVersion/graphviz-$portVersion.tar.bz2"
+CHECKSUM_SHA256="c84e17cee66500aae992ffad05a218f6a9619c59d15198f2a9f26d39cebf2bfc"
 SOURCE_FILENAME="graphviz-$portVersion.tar.gz"
```
Comment (korli) [inline media-gfx/graphviz/graphviz-2.49.1.recipe:20]: is SOURCE_FILENAME needed?

Comment (Begasus) [reply media-gfx/graphviz/graphviz-2.49.1.recipe:20]: Good catch :)

File: media-gfx/graphviz/patches/graphviz-2.49.1.patchset [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+ 	UWIN=yes
+@@ -185,6 +186,16 @@ case "${host_os}" in
+ 		;;
+         esac
+ 	;;
++	*haiku* )
++	HAIKU=yes
++	HAIKU_SYSTEM_FONTPATH=`finddir B_SYSTEM_FONTS_DIRECTORY`
++	HAIKU_COMMON_FONTPATH=`finddir B_COMMON_FONTS_DIRECTORY`
++	HAIKU_USER_FONTPATH=`finddir B_USER_FONTS_DIRECTORY`
```
Comment (korli) [inline media-gfx/graphviz/patches/graphviz-2.49.1.patchset:27]: nowadays we have B_SYSTEM_FONTS_DIRECTORY B_SYSTEM_NONPACKAGED_FONTS_DIRECTORY B_USER_FONTS_DIRECTORY  B_USER_NONPACKAGED_FONTS_DIRECTORY

File: media-gfx/graphviz/graphviz-2.49.1.recipe (hunk 1 above)
Comment (Begasus) [reply media-gfx/graphviz/graphviz-2.49.1.recipe]: Fixed :)

File: media-gfx/graphviz/patches/graphviz-2.49.1.patchset (hunk 2 above)
Comment (Begasus) [reply media-gfx/graphviz/patches/graphviz-2.49.1.patchset:27]: `finddir -l` still gives a line for `B_COMMON_FONTS_DIRECTORY`, but I could change those

Comment (Begasus) [reply media-gfx/graphviz/patches/graphviz-2.49.1.patchset:27]: Something like:
```diff
+	HAIKU_SYSTEM_NONPACKAGED_FONTPATH=`finddir B_SYSTEM_NONPACKAGED_FONTS_DIRECTORY `
```

Comment (Begasus) [reply media-gfx/graphviz/patches/graphviz-2.49.1.patchset:27]: Just checked, both `B_COMMON_FONTS_DIRECTORY` and `B_SYSTEM_NONPACKAGED_FONTS_DIRECTORY` result in the same path: `/boot/system/non-packaged/data/fonts`

Comment (pulkomandy) [reply media-gfx/graphviz/patches/graphviz-2.49.1.patchset:27]: yes, but the "COMMON" constants should not be used anymore. And B_USER_NONPACKAGED_FONTS_DIRECTORY will still be missing.

Comment (Begasus) [reply media-gfx/graphviz/patches/graphviz-2.49.1.patchset:27]: Working on it already ;)

File: media-gfx/graphviz/patches/graphviz-2.49.1.patchset [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+   *uwin* )
+ 	UWIN=yes
+@@ -185,6 +186,17 @@ case "${host_os}" in
+ 		;;
+         esac
+ 	;;
++	*haiku* )
++	HAIKU=yes
++	HAIKU_SYSTEM_FONTPATH=`finddir B_SYSTEM_FONTS_DIRECTORY`
++	HAIKU_SYSTEM_NONPACKAGED_FONTPATH=`finddir B_SYSTEM_NONPACKAGED_FONTS_DIRECTORY`
```
Comment (Begasus) [inline media-gfx/graphviz/patches/graphviz-2.49.1.patchset:26]: @korli fixed here and for HAIKU_USER_NONPACKAGED_FONTPATH, could you check? Will give this another spin on the laptop

File: media-gfx/graphviz/patches/graphviz-2.49.1.patchset [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
++	HAIKU_USER_FONTPATH=`finddir B_USER_FONTS_DIRECTORY`
++	HAIKU_USER_NONPACKAGED_FONTPATH=`finddir B_USER_NONPACKAGED_FONTS_DIRECTORY`
++	DEFAULT_FONTPATH="$HAIKU_SYSTEM_FONTPATH:$HAIKU_SYSTEM_FONTPATH/ttfonts:$HAIKU_SYSTEM_FONTPATH/psfonts:$HAIKU_SYSTEM_NONPACKAGED_FONTPATH:$HAIKU_SYSTEM_NONPACKAGED_FONTPATH/ttfonts:$HAIKU_SYSTEM_NONPACKAGED_FONTPATH/psfonts:$HAIKU_USER_FONTPATH:$HAIKU_USER_FONTPATH/ttfonts:$HAIKU_USER_FONTPATH/psfonts:$HAIKU_USER_NONPACKAGED_FONTPATH:$HAIKU_USER_NONPACKAGED_FONTPATH/ttfonts:$HAIKU_USER_NONPACKAGED_FONTPATH/psfonts"
```
Comment (korli) [inline media-gfx/graphviz/patches/graphviz-2.49.1.patchset:29]: could be split up (untested patch)
```suggestion
+	DEFAULT_FONTPATH="$HAIKU_SYSTEM_FONTPATH:$HAIKU_SYSTEM_FONTPATH/ttfonts:$HAIKU_SYSTEM_FONTPATH/psfonts"
+	DEFAULT_FONTPATH="$DEFAULT_FONTPATH:$HAIKU_SYSTEM_NONPACKAGED_FONTPATH:$HAIKU_SYSTEM_NONPACKAGED_FONTPATH/ttfonts:$HAIKU_SYSTEM_NONPACKAGED_FONTPATH/psfonts"
+	DEFAULT_FONTPATH="$DEFAULT_FONTPATH:$HAIKU_USER_FONTPATH:$HAIKU_USER_FONTPATH/ttfonts:$HAIKU_USER_FONTPATH/psfonts"
+	DEFAULT_FONTPATH="$DEFAULT_FONTPATH:$HAIKU_USER_NONPACKAGED_FONTPATH:$HAIKU_USER_NONPACKAGED_FONTPATH/ttfonts:$HAIKU_USER_NONPACKAGED_FONTPATH/psfonts"
```

File: media-gfx/graphviz/patches/graphviz-2.49.1.patchset (hunk 3 above)
Comment (korli) [reply media-gfx/graphviz/patches/graphviz-2.49.1.patchset:26]: @Begasus correct. I would split up the definition for DEFAULT_FONTPATH :)

File: media-gfx/graphviz/patches/graphviz-2.49.1.patchset (hunk 4 above)
Comment (Begasus) [reply media-gfx/graphviz/patches/graphviz-2.49.1.patchset:29]: Was thinking about that too, wasn't sure how to though

Comment (Begasus) [reply media-gfx/graphviz/patches/graphviz-2.49.1.patchset:29]: That could work, just need to re-create the patchset for it then

Comment (Begasus) [reply media-gfx/graphviz/patches/graphviz-2.49.1.patchset]: patchset updated, please check (build is ok on 32bit)

**Comment by korli** [comment]
Looks good to me.

**Pull request merged**